content/en/docs/ops/integrations/spire/index.md

```diff
@@ -19,188 +19,143 @@ SPIRE's node attestation extends attestation to the physical or virtual hardware
 
 For a quick demo of how this SPIRE integration with Istio works, see [Integrating SPIRE as a CA through Envoy's SDS API]({{< github_tree >}}/samples/security/spire).
 
-{{< warning >}}
-Note that this integration requires version 1.14+ for both `istioctl` and the data plane.
-{{< /warning >}}
-
-The integration is compatible with Istio upgrades.
-
 ## Install SPIRE
 
-### Option 1: Quick start
+We recommend you follow SPIRE's installation instructions and best practices for installing SPIRE, and for deploying SPIRE in production environments.
 
-Istio provides a basic sample installation to quickly get SPIRE up and running:
+For the examples in this guide, the [SPIRE Helm charts](https://artifacthub.io/packages/helm/spiffe/spire) will be used with upstream defaults, to focus on just the configuration necessary to integrate SPIRE and Istio.
 
-{{< text syntax=bash snip_id=install_spire_with_controller_manager >}}
-$ kubectl apply -f @samples/security/spire/spire-quickstart.yaml@
+{{< text syntax=bash snip_id=install_spire_crds >}}
+$ helm upgrade --install -n spire-server spire-crds spire-crds --repo https://spiffe.github.io/helm-charts-hardened/ --create-namespace
 {{< /text >}}
 
-This will deploy SPIRE into your cluster, along with two additional components: the [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi) — used to share the SPIRE Agent's UNIX Domain Socket with the other
-pods throughout the node — and the [SPIRE Controller Manager](https://github.com/spiffe/spire-controller-manager), a facilitator that performs workload registration and establishes federation relationships
-within Kubernetes. See [Install Istio](#install-istio) to configure Istio and integrate with the SPIFFE CSI Driver.
+{{< text syntax=bash snip_id=install_spire_istio_overrides >}}
+$ helm upgrade --install -n spire-server spire spire --repo https://spiffe.github.io/helm-charts-hardened/ --wait -f - <<EOF
+global:
+  spire:
+    trustDomain: example.org
+EOF
+{{< /text >}}
 
-### Option 2: Configure a custom SPIRE installation
+{{< tip >}}
+See the [SPIRE Helm chart](https://artifacthub.io/packages/helm/spiffe/spire) documentation for other values you can configure for your installation.
+{{< /tip >}}
 
-See the [SPIRE's Quick start for Kubernetes guide](https://spiffe.io/docs/latest/try/getting-started-k8s/)
-to get started deploying SPIRE into your Kubernetes environment. See [SPIRE CA Integration Prerequisites](#spire-ca-integration-prerequisites)
-for more information on configuring SPIRE to integrate with Istio deployments.
+It is important that SPIRE and Istio are configured with the exact same trust domain, to prevent authentication and authorization errors.
```

Comment:
this comment seems a little lost here; I assume it's the reason you are setting example.org in the option above?  Should it come before it if so?

Reply:
Rereading it, it really belongs in the `{{ tip }}`, so I moved it there.